File: Habitica/src/main/java/com/magicmicky/habitrpgwrapper/lib/models/invitations/PartyInvite.java
package com.magicmicky.habitrpgwrapper.lib.models.invitations;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by keithholliday on 7/2/16.
 */
public class PartyInvite {
    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("inviter")
    @Expose
    private String inviter;

    /**
     * @return The id
     */
    public String getId() {
        return id;
    }

    /**
     * @param id The id
     */
    public void setId(String id) {
        this.id = id;
    }

    /**
     * @return The name
     */
    public String getName() {
        return name;
    }

    /**
     * @param name The name
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * @return The inviter
     */
    public String getInviter() {
        return inviter;
    }

    /**
     * @param inviter The inviter
     */
    public void setInviter(String inviter) {
        this.inviter = inviter;
    }
}

File: shared/src/commonMain/kotlin/com/habitrpg/shared/habitica/models/invitations/PartyInvite.kt
package com.habitrpg.shared.habitica.models.invitations

import com.habitrpg.shared.habitica.nativePackages.NativeRealmObject
import com.habitrpg.shared.habitica.nativePackages.annotations.PrimaryKeyAnnotation

open class PartyInvite : NativeRealmObject(), GenericInvitation {

    @PrimaryKeyAnnotation
    override var id: String? = null
    override var name: String? = null
    override var inviter: String? = null
}
